app/routes.js

```diff
@@ -26,7 +26,7 @@ export default function createRoutes(store) {
   return [
     {
       path: '/',
-      name: 'Home',
+      name: ' ',
```

Comment:
@marcmos czy to jedyny sposób żeby pozbyć się napisu Home z AppBar albo przynajmniej czy słuszny / wystarczający / niewadzący